src/features/unified-share-modal/SharedLinkSection.js

```diff
@@ -384,7 +385,21 @@ class SharedLinkSection extends React.Component<Props, State> {
                         <span className="security-indicator-icon-globe">
                             <IconGlobe height={12} width={12} />
                         </span>
-                        {permissionLevel === CAN_EDIT ? (
+                        // TODO: temporary change to support Canvas not being truly public
+                        {/* eslint-disable-next-line no-nested-ternary */}
+                        {isBoxCanvasFile ? (
+                            permissionLevel === CAN_EDIT ? (
+                                <FormattedMessage
+                                    data-testid="shared-link-editable-publicly-available-message"
+                                    {...messages.canvasSharedLinkEditablePubliclyAvailable}
+                                />
+                            ) : (
+                                <FormattedMessage
+                                    data-testid="shared-link-publicly-available-message"
+                                    {...messages.canvasSharedLinkPubliclyAvailable}
+                                />
+                            )
+                        ) : permissionLevel === CAN_EDIT ? (
```

Comment:
why does this need separate messages? it has the same output for any Box Canvas regardless of the permission level right?

Reply:
You're right, thanks for pointing that out!